libs/cdk/render-strategies/src/lib/concurrent-strategies.ts

```diff
@@ -126,6 +169,39 @@ function scheduleOnQueue<T>(
       )
     );
 }
+function scheduleOnPostTaskQueue<T>(
+  work: (...args: any[]) => void,
```

Comment:
Just wondering,

I've noticed the usage on `any` in many libraries, whilst Sonarqube scans recommend to replace `any` with `unknown` where possible. Sometimes it causes too strict typing and `any` could be preferred.

When I have a first look at this code, it seems that `any` could be replaced here with `unknown`. I'm not saying it's wrong however I wonder how serious this rule should be taken.

What is your opinion on this?

Reply:
agree, `unknown` is better than `any`. This would in best case be done as part of this effort: https://github.com/rx-angular/rx-angular/issues/976 :)